### PR TITLE
invalid 'sytle' tag

### PR DESCRIPTION
**Pull request by mobyte0**
¯\\_(ツ)_/¯

### CODE REVIEW

**Comment by scott-linder** [comment]
Looks good! Thank you for working on the site, I think it needs the work.

Sorry it took me a bit to check it out, I had to install jekyll on my new laptop.

File: index.html [hunk 1]
```diff
@@ -5,44 +5,45 @@
 {% include header.html %}
 
 <iframe style="margin-top: 1px;" width="100%" height="500" frameborder="0"
-    src="https://www.flickr.com/photos/129058274@N06/sets/72157648511414994/player"></iframe>
+        src="https://www.flickr.com/photos/129058274@N06/sets/72157648511414994/player"></iframe>
 
 <div class="content">
-    <div sytle="text-align: left;" class="c-box">
+    <div style="text-align: center;" class="c-box">
```
Comment (scott-linder) [inline index.html]: I think this entire `style` attr can just go away; the `c-box` class already centers it.

**Comment by jpypi** [comment]
Looks good to me. Thanks for noticing and updating these things!

**Comment by jpypi** [comment]
Looks good! Thanks for noticing and updating!

**Pull request merged**